Source/Dafny/Resolver.cs

```diff
@@ -14140,28 +14140,36 @@ class LazyString_OnTypeEquals
     /// "context == Co" says that "type" is allowed to vary in the positive direction.
     /// "context == Contra" says that "type" is allowed to vary in the negative direction.
     /// "context == Inv" says that "type" must not vary at all.
-    /// * "leftOfArrow" says that the context is to the left of some arrow
+    /// * "lax" says that the context is not strict -- type parameters declared to be strict must not be used in a lax context
     /// </summary>
-    public void CheckVariance(Type type, ICallable enclosingTypeDefinition, TypeParameter.TPVariance context, bool leftOfArrow) {
+    public void CheckVariance(Type type, ICallable enclosingTypeDefinition, TypeParameter.TPVariance context, bool lax) {
       Contract.Requires(type != null);
       Contract.Requires(enclosingTypeDefinition != null);
 
       type = type.Normalize();  // we keep constraints, since subset types have their own type-parameter variance specifications; we also keep synonys, since that gives rise to better error messages
+
```

Comment:
This and L14187 below make me think your IDE is set to insert blank lines before `if` statements or blocks. If you didn't intentionally insert this line, please adjust your IDE style formatting settings.